toolchain/sem_ir/name_scope_test.cpp

```diff
@@ -265,27 +317,31 @@ TEST(NameScope, Poison) {
   NameScope name_scope(scope_inst_id, scope_name_id, parent_scope_id);
 
   NameId poison1(++id);
-  EXPECT_EQ(name_scope.LookupOrPoison(poison1), std::nullopt);
-  EXPECT_THAT(
-      name_scope.entries(),
-      ElementsAre(NameScope::Entry(
-          {.name_id = poison1, .result = ScopeLookupResult::MakePoisoned()})));
+  LocId poisoning1(++id);
```

Comment:
I feel like this would be more clear and consistent:
```suggestion
  LocId poisoning_loc1(++id);
```
or
```suggestion
  LocId poisoning_loc_id1(++id);
```

Reply:
Done.